src/CoreBundle/CronJob/CronJobFactory.php

```diff
@@ -23,24 +18,19 @@ public function __construct(ContainerInterface $container)
 
     /**
      * @param string[] $idList
+     *
+     * @deprecated since 6.3.0 - available cron jobs are injected by constructor now
```

Comment:
Wikipedia calls it "cron job" but the majority of the internet uses "cronjob" (?)

Reply:
Both notations are common, so I decided for "cron job" in newer code which seems more "correct" to me.